frontend/components/radialProfileCard.tsx

```diff
@@ -15,54 +16,118 @@ import {
   DialogHeader,
   DialogTitle,
   DialogTrigger,
-} from '@/components/ui/dialog';
-
-import { HoverCard, HoverCardContent, HoverCardTrigger } from '@/components/ui/hover-card';
-
+} from "@/components/ui/dialog";
 import { Button } from '@/components/ui/button';
 import Image from 'next/image';
 
+// get API endpoint BE
+export const fetchUserData = async () => {
+  try {
```

Comment:
This should be removed? It is used in utils profileCompletionCheck.ts I see?

Reply:
ohh profileCompletionCheck.ts  is a redundant file it's not needed I forgot to delete that file

Reply:
No have it there, remove this @Lanthoiba2022

Reply:
ohh Ok I'll make the changes